spec/migrations/encryptables_data_structure_spec.rb

```diff
@@ -0,0 +1,135 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+mig_file = Dir[Rails.root.join('db/migrate/20220104140658_use_encrypted_data_for_account_credentials.rb')].first
+require mig_file
+
+describe UseEncryptedDataForAccountCredentials do
+
+  let(:migration) { described_class.new }
+
+  let(:folder1) {folders(:folder1)}
+
+  let(:account1) {accounts(:account1)}
+  let(:account2) {accounts(:account2)}
+  let!(:account3) do
+    Account::Credentials.create!(name: 'spacex', folder: folder1, encrypted_data: {
+      password: {data: '', iv: nil},
```

Comment:
we do not need an account with blank values inside encrypted data.
we need a LegacyAccount::Credentials with blank/nil values in :password, :username, since this is what exists today and we wanna make sure this gets migrated correctly